easybuild/easyconfigs/e/expat/expat-2.2.6-GCCcore-8.2.0.eb

```diff
@@ -18,6 +18,11 @@ source_urls = [SOURCEFORGE_SOURCE]
 sources = [SOURCELOWER_TAR_BZ2]
 checksums = ['17b43c2716d521369f82fc2dc70f359860e90fa440bea65b3b85f0b246ea81f2']
 
+# since expat 2.2.6, docbook2X is needed to produce manpage of xmlwf
+# docbook2X needs XML-Parser and XML-Parser needs expat
+# so circular dependency. "--without-docbook" breaks this circle.
+configopts = ['--without-docbook']
+
```

Comment:
configopts go below builddeps, please fix asap, we need this for some other PR's deps

Reply:
done.